### PR TITLE
fix(central): add incoming ticket too

### PR DESCRIPTION
**Pull request by stonebuzz**
GLPI allows you to assign a group to a ticket and leave the status as new.

But the central page -> group view -> tickets to be processed 

does not take into account the new status

This PR fix this

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | internal 21256

### CODE REVIEW

File: inc/ticket.class.php [hunk 1]
```diff
@@ -5331,15 +5333,22 @@ static function showCentralList($start, $status = "process", $showgrouptickets =
...
+                  $options['criteria'][4]['link']                       = 'AND';
+                  $options['criteria'][4]['criteria'][0]['link']        = 'AND';
+                  $options['criteria'][4]['criteria'][0]['field']       = 12;
+                  $options['criteria'][4]['criteria'][0]['searchtype']  = 'equals';
+                  $options['criteria'][4]['criteria'][0]['value']       = 1;
+
+                  $options['criteria'][4]['criteria'][1]['link']        = 'OR';
+                  $options['criteria'][4]['criteria'][1]['field']       = 12;
+                  $options['criteria'][4]['criteria'][1]['searchtype']  = 'equals';
+                  $options['criteria'][4]['criteria'][1]['value']       = 'process';
```
Comment (cedric-anne) [inline inc/ticket.class.php:5351]: Why using indexes 2 and 4 ?
Could it be possible to have something like
```php
$options['criteria'] = [
   [
      'field' => 8
      ...
   ],
   [
      'link' => 'AND',
      'criteria' => [
         [
            'field' => 12,
            ...
         ],
         [
            'field' => 12,
            ...
         ]
      ]
   ]
];
```

Also, position of AND/OR links seems weird to me, but maybe it is done like this in search class.

File: inc/ticket.class.php [hunk 2]
```diff
@@ -5100,12 +5100,13 @@ static function showCentralList($start, $status = "process", $showgrouptickets =
             );
             break;
 
-         case "process" : // planned or assigned tickets
+         case "process" : // planned or assigned or incoming tickets
             $WHERE = array_merge(
                $WHERE,
                $search_assign,
-               ['glpi_tickets.status' => self::getProcessStatusArray()]
```
Comment (trasher) [inline inc/ticket.class.php:5107]: I'd prefer to keep the call to the `getProcessStatusArray`, in case it change in the future.

**Review by trasher — COMMENTED** [review]
Seems ok from a technical pov (only see my comment), but no real idea on a functionnal pov :D

File: inc/ticket.class.php (hunk 2 above)
Comment (stonebuzz) [reply inc/ticket.class.php:5107]: Fixed

File: inc/ticket.class.php (hunk 1 above)
Comment (stonebuzz) [reply inc/ticket.class.php]: Fixed by using new convention

**Review by trasher — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**